draft-ietf-rtcweb-jsep.xml

```diff
@@ -694,35 +692,18 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
         <section title="Creating an imageattr Attribute">
 
-          <t>In order to determine the limits on what video resolution
-          a receiver wants to receive, it will intersect its decoder
-          hard limits with any mandatory constraints that have been
-          applied to the associated MediaStreamTrack. If the decoder
-          limits are unknown, e.g. when using a software decoder, the
-          mandatory constraints are used directly. For the answerer,
-          these mandatory constraints can be applied to the remote
-          MediaStreamTracks that are created by a setRemoteDescription
-          call, and will affect the output of the ensuing createAnswer
-          call. Any constraints set after setLocalDescription is used
-          to set the answer will result in a new offer-answer exchange.
-          For the offerer, because it does not know about any remote
-          MediaStreamTracks until it receives the answer, the offer can
-          only reflect decoder hard limits. If the offerer wishes to
-          set mandatory constraints on video resolution, it must do so
-          after receiving the answer, and the result will be a new
-          offer-answer to communicate them.</t>
-
-          <t>If there are no known decoder limits or mandatory
-          constraints, the "a=imageattr" attribute SHOULD be
-          omitted.</t>
+          <t>The receiver will first intersect any known local limits (e.g.,
```

Comment:
I think given that this is an "e.g.", "merge" might be a happier word than "intersect" but I can live with this.

I note that you have now removed the viewport, so that if you attach to a 32x32 window you may still get 1080p. Is that what we agreed.